src/clients/SpokePoolClient.ts

```diff
@@ -364,7 +364,7 @@ export class SpokePoolClient extends BaseAbstractClient {
    * @returns A valid fill for the deposit, or undefined.
    */
   public getFillForDeposit(deposit: Deposit): FillWithBlock | undefined {
```

Comment:
If you have time can you add a very simple unit test here you create two fills for same `getDepositHash()` and test that it returns > 1 ? Just for regression prevention

Reply:
https://github.com/across-protocol/sdk/pull/869/commits/e87700f265e1f94288c251d7f91d7afe81e287fc